<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VotingPage</class>
 <widget class="QWidget" name="VotingPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Voting</string>
  </property>
  <layout class="QVBoxLayout" name="votingPageVerticalLayout" stretch="0,0,0,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item alignment="Qt::AlignVCenter">
    <widget class="QFrame" name="headerFrame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout" name="headerFrameLayout">
      <property name="spacing">
       <number>15</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignVCenter">
       <widget class="QWidget" name="headerTitleWrapper" native="true">
        <layout class="QVBoxLayout" name="headerTitleVerticalLayout">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="headerTitleLabel">
           <property name="text">
            <string>Polls</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="headerFrameSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="filterLineEdit">
        <property name="placeholderText">
         <string>Search by title</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="tabButtonFrame">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>9</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="cardsToggleButton">
        <property name="toolTip">
         <string>View as list.</string>
        </property>
        <property name="shortcut">
         <string>Alt+T</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tableToggleButton">
        <property name="toolTip">
         <string>View as table.</string>
        </property>
        <property name="shortcut">
         <string>Alt+T</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="sortButton">
        <property name="toolTip">
         <string>Sort by...</string>
        </property>
        <property name="shortcut">
         <string>Alt+S</string>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignVCenter">
       <widget class="QPushButton" name="refreshButton">
        <property name="text">
         <string>&amp;Refresh</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignVCenter">
       <widget class="QPushButton" name="createPollButton">
        <property name="text">
         <string>Create &amp;Poll</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="pollReceivedLabel">
     <property name="text">
      <string>A new poll is available. Press &quot;Refresh&quot; to load it.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="tabWrapperWidget" native="true">
     <layout class="QVBoxLayout" name="tabWrapperWidgetLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="PollTab" name="activePollsTab">
         <attribute name="title">
          <string>&amp;Active</string>
         </attribute>
        </widget>
        <widget class="PollTab" name="finishedPollsTab">
         <attribute name="title">
          <string>&amp;Completed</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PollTab</class>
   <extends>QWidget</extends>
   <header>voting/polltab.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
